src/pages/Transaction/TransactionComp.tsx

```diff
@@ -327,6 +331,78 @@ const handleCellbaseInputs = (inputs: State.Cell[], outputs: State.Cell[]) => {
   return inputs
 }
 
+export const TransactionCompLite: FC<{ transaction: State.Transaction }> = ({ transaction }) => {
+  const { hash: txHash } = useParams<{ hash: string }>()
+  const { isCellbase } = transaction
+
+  const query = useQuery(['ckb_transaction_details', txHash], async () => {
+    const ckbTransactionDetails = await fetchTransactionLiteDetailsByHash(txHash)
+    return ckbTransactionDetails.data
+  })
+
+  const transactionLiteDetails: State.TransactionLiteDetails[] = query.data ?? defaultTransactionLiteDetails
+
+  return (
+    <>
+      {transactionLiteDetails &&
+        transactionLiteDetails.map(item => (
+          <div className="transaction_lite" key={item.address}>
+            <div className={styles.transactionLiteBox}>
+              <div className={styles.transactionLiteBoxHeader}>
+                <div className={styles.transactionLiteBoxHeaderAddr}>
+                  <Addr address={item.address} isCellBase={isCellbase} />
+                </div>
+              </div>
+              <div className={styles.transactionLiteBoxContent}>
+                {item.transfers.map((transfer, index) => {
+                  return (
+                    <div key={`transfer-${index}`}>
```

Comment:
It still relies on `index`. Does `transfer` has its own id, e.g. `tokenId`. I see `transfer.nftId` is mentioned below

Reply:
I think none of its attribute is unique: https://github.com/Magickbase/ckb-explorer-frontend/blob/transaction-lite/src/types/index.d.ts#L140-L152

Reply:
I see